source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -56,7 +56,7 @@ Http::FilterHeadersStatus CacheFilter::decodeHeaders(Http::RequestHeaderMap& hea
         *decoder_callbacks_, headers);
     return Http::FilterHeadersStatus::Continue;
   }
-  if (!CacheabilityUtils::isCacheableRequest(headers)) {
```

Comment:
@ekiziv can you add a version history entry for this feature in https://www.envoyproxy.io/docs/envoy/latest/version_history/current? Thanks. LGTM otherwise.